index.html

```diff
@@ -19,14 +19,123 @@ <h1 class="main-text">I'M ZALIHA GLAD TO SEE YOU</h1>
 				<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
 			</section>
 			<a href="#" class="connect">LET'S CONNECT</a>
-					<div class="icons">
-							<img src="./images/twitter.png" alt="">
-							<img src="./images/Linkedin.png" alt="">
-							<img src="./images/vector-m.png" alt="">
-							<img src="./images/baby.svg" alt="">
-							<img src="./images/hand.png" alt="">
-							
+				<div class="icons">
+						<img src="./images/twitter.png" alt="">
+						<img src="./images/Linkedin.png" alt="">
+						<img src="./images/vector-m.png" alt="">
+						<img src="./images/baby.svg" alt="">
+						<img src="./images/hand.png" alt="">
+						
+				</div>
+				<section class="works">
+					<div class="works-section">
+						<div>
+							<img class=image-work src="./images/portfolio1.png" alt="">
+							<h1>Tonic</h1>
+							<span class="ball">Canopy</span>
+							<span>Backend</span>
+							<span>2015</span>
+						</div>
+						<div>
+							<p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+								
+									<span class="stack">Html</span>
+									<span class="stack">Css</span>
+									<span class="stack">Javascript</span>
+								</div>
```

Comment:
Instead of` span` tags, kindly use a list `ul` with `li` elements for all your project technologies.

- [ ] Use semantic tags for your project technologies in the "Works" section, do the same for each project.